### DISCUSSION TITLE
Change process English source strings via Weblate

### DISCUSSION THREAD

**Discussion by HaaiHenkie** [freeplane/freeplane#2067] [General]
Hello Freeplane community,

A few days ago @htgoebel changed some English source strings through Weblate. I haven't looked at them in detail, but on first sight they are OK changes. The question I have is: does anybody review this before it is changed in the source code and finally released in a new version of Freeplane? I believe the quality of the source strings is very important. to help people understand menus and options in Freeplane in English and for the translators to provide a translation of the same quality. Changes of  source strings in Weblate could increase the quality, but also decrease the quality and they could even be wrong if the person changing the source strings does not fully understand Freeplane's functionality.

Quite some years ago I wrote a topic on [changing English source strings](https://docs.freeplane.org/contribute/Translation_How-To.html#change-english-source-strings) which is still in the current user guide. My suggestion was to discuss changes of source strings on the forum first. It still refers to the old forum, but I could change that. Do you think that is an OK approach?

I think on Weblate it is also possible to review changed source strings before they are submitted. Would that be a better approach?

Best regards,

Henk

**Comment by dpolivaev** [comment]
Do  you mean changes introduced by @htgoebel in [commit 821725b683ec5064c6808cf4fa71921fcbc7a082](https://github.com/freeplane/freeplane/commit/821725b683ec5064c6808cf4fa71921fcbc7a082)? 

Actually it just replaces -> by arrow: the \u2192 is the Unicode symbol for a right arrow (→). All other changes were done by me. And actually I am not opposed to improvements done by others in English strings, and if they are obvious, I do not think we always need to have an extra discussion.

When the texts were not clear enough, Hartmut wrote comments and I decided to improve the texts.

**Comment by HaaiHenkie** [comment]
You are right I got a bit confused between his comments and the changes you made. I agree people should improve obvious mistakes like spelling errors. I assume you review when people do more drastic changes? I do not think it ever happens though.

While translating I noticed a small mistake in IconProgressIconUpAction.tooltip:

![image](https://github.com/user-attachments/assets/9c254d37-842f-4439-8fdd-eb3c80de6b56)

The plus should have remained since the last increment is to 100% and the OK icon is added

**Reply by dpolivaev** [reply]
No, I actually usually do not review any changes in translations including the English.

**Comment by htgoebel** [comment]
Am 05.10.24 um 16:12 schrieb Dimitry Polivaev:
> I decided to improve the texts

Thanks for the quick reaction :-)

I hope I did not annoy you with too many comment mails sent to you while 
I fixed the typos in the comments :-)

-- 
Regards
Hartmut Goebel

| Hartmut Goebel          | ***@***.***               |
| www.crazy-compilers.com | compilers which you thought are impossible |
